src/components/previewCard/index.tsx

```diff
@@ -11,6 +11,7 @@ interface Props {
   cardTitle?: string
   previewLabel: string
   image?: string
+  loading?: "lazy" | "eager"
```

Comment:
I think `loading` is a bit too generic when we talk about the whole card. I'd expect the whole component to be lazy-loaded, not just the image. How about:

```suggestion
  imageLoading?: "lazy" | "eager"
```
?

Reply:
I agree, will change it, thanks!